wagtail_streamfield_migration_toolkit/test/tests/test_rename.py

```diff
@@ -0,0 +1,153 @@
+from django.test import TestCase, SimpleTestCase
+
+from wagtail_streamfield_migration_toolkit.utils import get_altered_raw_data
+from ..models import SampleModel
+
+
+class RenameUtilsTestCase(SimpleTestCase):
+    @staticmethod
+    def make_json_data():
+        stream_data = [
+            {
+                "type": "field1",
+                "value": "Simple Rename Value",
+            },
+            {
+                "type": "field2",
+                "value": "This shouldn't change",
+            },
+            {
+                "type": "struct1",
+                "value": {
+                    "struct1field1": "Struct field rename",
+                    "struct1field2": "This shouldn't change",
+                },
+            },
+            {
+                "type": "stream1",
+                "value": [
+                    {"type": "stream1field1", "value": "Nested Stream field rename"},
+                    {"type": "stream1field2", "value": "This shouldn't change"},
+                ],
+            },
+        ]
+        return stream_data
+
+
+class RenameRawDataTestCase(TestCase):
+    """
+    Tests without saving or loading from the database.
+    This is to test just the changes to the raw json data, without considering the schema changes,
+    or anything post save.
+    """
+
+    @staticmethod
+    def create_sample_page():
+        sample_page = SampleModel(
```

Comment:
Given that this model isn't technically a Wagtail `Page`, it might be a little confusing to call it `sample_page`